Dockerfile

```diff
@@ -112,3 +113,4 @@ LABEL \
     org.opencontainers.image.title="ddns-updater" \
     org.opencontainers.image.description="Universal DNS updater with WebUI"
 COPY --from=build --chown=${UID}:${GID} /tmp/gobuild/app /updater/app
+COPY --from=build --chown=${UID}:${GID} /tmp/data /updater/data

```

Comment:
Same here, can you move this line right after `ENTRYPOINT ["/updater/app"]` since it shouldn't change really (unlike `ENV` / `ARG` etc.)

Reply:
Done